packages/ui/src/components/Accordion/MetaDataStatusItem.tsx

```diff
@@ -1,28 +1,50 @@
+/* eslint-disable camelcase */
```

Comment:
Could you put the exception just with the single line?

Reply:
Moved in [move eslint ignore to line](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13657/commits/9e6eb4c83466a64f878651bad8c1fa39ab0e18e3)